ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2165,15 +2166,15 @@ <h3 style="margin-top: 1.2em;">Sprint</h3>
 
     <div class="span8 offset2">
       <h2>Being part of PLOG is easy.</h2>
-      <p>PLOG will take place <a href="https://plone.org/events/sprints/sorrento-sprint-on-frontend-modernization">April 12 to 19, 2019</a>, in Sorrento, Italy. Admission to PLOG is free and open to everyone
-          who wants to be in touch with Plone’s community.
+      <p>PLOG will take place <a href="https://plone.org/events/sprints/plog-2020">April 19 to 26, 2020</a>, in Sorrento, Italy. 
+	  Admission to PLOG is free and open to everyone who wants to be in touch with Plone’s community.
           You can come alone or together with your friends and family, snatching the opportunity to visit beautiful
           Sorrento.
-          Being part of it is simple: just sign up by filling out the form below.
+          Being part of it is simple: just sign up by filling out the registration form.
       </p>
         <div class="row-fluid">
             <div class="span8 offset4 text-right">
-                <a class="btn btn-flat btn-primary" href="/plog-registration-form">
+                <a class="btn btn-flat btn-primary" href="/plog-registration-form-2020">
```

Comment:
I thought a good reason for leaving the URL pointing to a generic (non-year-dated) form is that you could update the form and not have to update this page.

Reply:
@tkimnguyen what you say makes total sense, and I am sooooo kicking myself that I did it this way. It started with me feeling uncertain about making the edits, so I made a copy to work on and named it xxx-2020. I was thinking that when it was ready I could rename/remove the old one and rename the new one to plog-registration. But upon reflection I decided that wouldn't work, because Plone would know that the old one moved and would continue pointing to it. That's why I decided to keep it this way for this year, and go back to the generic name next year.

But maybe there is a way to get the rename/swap to work, which would be better. I'll look into that.

Reply:
I'd say just go ahead and make the live edits to the existing form

Reply:
Done, plog-registration-form has been updated and this page now points to that URL.